Source/Dialogs/Dialogs.cpp

```diff
@@ -426,14 +426,8 @@ void Dialogs::showCanvasRightClickMenu(Canvas* cnv, Component* originalComponent
             }
         };
 
-        std::unique_ptr<CheckedTooltip> tooltipWindow;

```

Comment:
This adds another tooltip window, when semitransparent is supported.

However on w11 - this is unnecessary as so far with my testing it looks to work fine in this situation.